localization/yabloc/yabloc_common/include/yabloc_common/timer.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef YABLOC_COMMON__TIMER_HPP_
+#define YABLOC_COMMON__TIMER_HPP_
+
+#include <chrono>
+#include <ostream>
+namespace yabloc::common
+{
+class Timer
```

Comment:
maybe, it is better to use [this](https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_autoware_utils/include/tier4_autoware_utils/system/stop_watch.hpp#L27) pray

Reply:
Fixed in  c744ff0 & da88235
I removed `yabloc_common/timer.hpp`